src/components/application_manager/test/mobile_message_handler_test.cc

```diff
@@ -214,7 +218,8 @@ class MobileMessageHandlerTest : public testing::Test {
   void TestHandlingOutgoingMessageProtocolWithBinaryData(
       const uint32_t protocol_version) {
     // Arrange
-    BinaryData* bin_dat = new BinaryData;
+    application_manager::BinaryData* bin_dat =
+        new application_manager::BinaryData;
```

Comment:
@dev-gh , this cannot be shared ptr because bin_dat will be deletet with message.